src/libecalc/core/models/compressor/results.py

```diff
@@ -290,15 +290,21 @@ def set_failure_status(cls, v, values):
 
     @property
     def chart_area_status(self) -> ChartAreaFlag:
-        chart_area_failure = [
-            stage.chart_area_flag
-            for stage in self.stage_results
-            if stage.chart_area_flag != ChartAreaFlag.INTERNAL_POINT
-        ]
+        """Checks where the operational points are placed in relation to the compressor charts in a compressor train.
```

Comment:
Set up docstring in PyCharm to use "Google" in "File > Settings > Tools > Python Integrated Tools > Docstring format" --> Google :)